package.json

```diff
@@ -20,7 +20,11 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "release": "standard-version",
+    "postrelease": "gh-pages -d build",
```

Comment:
Why is this command the same as deploy?

Reply:
good catch, I didn't mean to leave `deploy` in